config/locales/en.yml

```diff
@@ -138,23 +138,7 @@ en:
 
           - could help you complete a task but is not the main piece of content included in the step by step.
           - is optional and not useful enough to be included in the step by step
-    change_notes:
```

Comment:
Is the change notes guidance no longer being used?

Reply:
The ones I deleted are not used anywhere. 
The views use those ones and they are only used once:
https://github.com/alphagov/collections-publisher/blob/839e95f12a5d00f584f0556615ef5a4ae4d70d2f/config/locales/en.yml#L37-L59

Reply:
Can you add that to the commit message?